src/merge_stardist_masks/naive_fusion.py

```diff
@@ -597,13 +595,11 @@ def naive_fusion_anisotropic_grid(
                 > 0
             )
 
-            size_of_current_shape = np.sum(new_shape)
-
             new_shape = np.logical_or(
                 new_shape,
                 additional_shape,
             )
-            if size_of_current_shape == np.sum(new_shape):
+            if np.all(new_shape):
```

Comment:
The intent of this lines is to get the size of the `new_shape` before and after merging it with the `additional_shape`. Therefore, it is correct to use `np.sum`. I do not think that you can change that to `np.all`

Reply:
Good catch! You are right, I thought that all values in `new_shape` are `True` by default and that `np.sum(new_shape)` is just a strange way to calculate `new_shape.size`. I will revert the change tomorrow morning.